praw/models/reddit/subreddit.py

```diff
@@ -1517,6 +1524,96 @@ def update_invite(self, redditor, permissions=None):
         self.subreddit._reddit.post(url, data=data)
 
 
+class Modmail(object):
```

Comment:
I was thinking the first three endpoints listed [here](https://www.reddit.com/dev/api/#section_modmail) — mark all read, conversation list, and creating a new conversation — would belong in this class, since they aren't tied to a particular conversation. Same for the endpoints that mark conversations read or unread (since they can handle many convos at once), and the one that gets unread counts.